tinyagentos/routes/project_invites.py

```
@@ -605,6 +605,22 @@ def _build_os_guide_markdown(
+def _scopes_as_list(raw) -> list:
+    """The store keeps scopes as a JSON-encoded TEXT column; the API contract
+    is a list. Parse defensively so a malformed row degrades to [] instead of
+    crashing every client that renders the list."""
+    if isinstance(raw, list):
+        return raw
+    if isinstance(raw, str):
+        try:
+            import json as _json
+            v = _json.loads(raw)
+            return v if isinstance(v, list) else []
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)